internal/kafka/command_replica_list.go

```diff
@@ -1,84 +1,56 @@
 package kafka
 
 import (
-	"net/http"
-
 	"github.com/spf13/cobra"
 
 	"github.com/confluentinc/kafka-rest-sdk-go/kafkarestv3"
 
 	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
 	"github.com/confluentinc/cli/v3/pkg/examples"
 	"github.com/confluentinc/cli/v3/pkg/kafkarest"
 	"github.com/confluentinc/cli/v3/pkg/output"
-	"github.com/confluentinc/cli/v3/pkg/utils"
 )
 
-type replicaHumanOut struct {
-	ClusterId          string `human:"Cluster"`
-	TopicName          string `human:"Topic Name"`
-	BrokerId           int32  `human:"Broker ID"`
-	PartitionId        int32  `human:"Partition ID"`
-	IsLeader           bool   `human:"Leader"`
-	IsObserver         bool   `human:"Observer"`
-	IsIsrEligible      bool   `human:"ISR Eligible"`
-	IsInIsr            bool   `human:"In ISR"`
-	IsCaughtUp         bool   `human:"Caught Up"`
-	LogStartOffset     int64  `human:"Log Start Offset"`
-	LogEndOffset       int64  `human:"Log End Offset"`
-	LastCaughtUpTimeMs string `human:"Last Caught Up Time (ms)"`
-	LastFetchTimeMs    string `human:"Last Fetch Time (ms)"`
-	LinkName           string `human:"Link Name"`
-}
-
-type replicaSerializedOut struct {
-	ClusterId          string `serialized:"cluster_id"`
-	TopicName          string `serialized:"topic_name"`
-	BrokerId           int32  `serialized:"broker_id"`
-	PartitionId        int32  `serialized:"partition_id"`
-	IsLeader           bool   `serialized:"is_leader"`
-	IsObserver         bool   `serialized:"is_observer"`
-	IsIsrEligible      bool   `serialized:"is_isr_eligible"`
-	IsInIsr            bool   `serialized:"is_in_isr"`
-	IsCaughtUp         bool   `serialized:"is_caught_up"`
-	LogStartOffset     int64  `serialized:"log_start_offset"`
-	LogEndOffset       int64  `serialized:"log_end_offset"`
-	LastCaughtUpTimeMs int64  `serialized:"last_caught_up_time_ms"`
-	LastFetchTimeMs    int64  `serialized:"last_fetch_time_ms"`
-	LinkName           string `serialized:"link_name"`
+type ReplicaOut struct {
+	ClusterId   string `human:"Cluster" serialized:"cluster_id"`
+	TopicName   string `human:"Topic Name" serialized:"topic_name"`
+	PartitionId int32  `human:"Partition ID" serialized:"partition_id"`
```

Comment:
It feels like we have a lot of inconsistency across various commands as to whether or not `Name` or `ID` should be appended to the end of the human and/or serialized outputs.

Is this something we want to change in v4, and if so, what's the style we want to follow? Personally I think I prefer to just not have `Name` or `ID` for either human or serialized output because it should be pretty obvious what they are.

Reply:
> Personally I think I prefer to just not have Name or ID for either human or serialized output because it should be pretty obvious what they are.

+1

Reply:
Made a ticket for this. I don't think I'll do it for this PR; I'll just come back and do it all in a single pass.